From fde9ba8bf36b61988638c92ea29caeab6ed7f1dc Mon Sep 17 00:00:00 2001
From: Stephen Kitt <skitt@debian.org>
Date: Sun, 24 Nov 2019 11:29:00 +0100
Subject: [PATCH 1/2] Revert "Allow strong dependencies on X font packages"

This reverts commit 6db3b01a13cf5cf7f5e0ea96049c8f51abdb9cfd, which
ignored the fact that X clients can still run on remote X servers, and
the fonts are provided by the latter.
---
 policy/ch-customized-programs.rst | 17 ++++++++---------
 1 file changed, 8 insertions(+), 9 deletions(-)

diff --git a/policy/ch-customized-programs.rst b/policy/ch-customized-programs.rst
index dfe6ce4..dbba4fc 100644
--- a/policy/ch-customized-programs.rst
+++ b/policy/ch-customized-programs.rst
@@ -380,10 +380,11 @@ themselves.
 1.  Fonts of any type supported by the X Window System must be in a
     separate binary package from any executables, libraries, or
     documentation (except that specific to the fonts shipped, such as
-    their license information). Packages which require one or more of
-    the fonts thus packaged should Depend on the font package; if the
-    fonts merely provide an enhancement, a Recommends or Suggests
-    relationship may be used.  [#]_
+    their license information). If one or more of the fonts so packaged
+    are necessary for proper operation of the package with which they
+    are associated the font package may be Recommended; if the fonts
+    merely provide an enhancement, a Suggests relationship may be used.
+    Packages must not Depend on font packages.  [#]_
 
 2.  BDF fonts must be converted to PCF fonts with the ``bdftopcf``
     utility (available in the ``xfonts-utils`` package, ``gzip``\ ped,
@@ -616,11 +617,9 @@ installed in ``/usr/share/man/man6``.
    Window System, however, must abide by this font policy.
 
 .. [#]
-   In the past, the X server could retrieve fonts from the local file
-   system or over the network from an X font server, so packages were
-   forbidden from declaring a Depends relationship with font
-   packages. This is no longer the case: the X font server shipped in
-   Debian no longer supports remote font retrieval.
+   This is because the X server may retrieve fonts from the local file
+   system or over the network from an X font server; the Debian package
+   system is empowered to deal only with the local file system.
 
 .. [#]
    Note that this mechanism is not the same as using app-defaults;
-- 
2.20.1

